kratos/geometries/nurbs_curve_on_surface_geometry.h

```diff
@@ -249,6 +249,17 @@ class NurbsCurveOnSurfaceGeometry : public Geometry<typename TSurfaceContainerPo
         this->SpansLocalSpace(rSpans, interval.GetT0(), interval.GetT1());
     }
 
+    /* @brief Provides intersections of the nurbs curve with the knots of the surface,
+     *         using the interval of this curve in the SBM case.
+     * @param vector of span intervals.
+     * @param index of chosen direction, for curves always 0.
+     */
+    void SpansLocalSpaceSBM(std::vector<double>& rSpans, IndexType DirectionIndex = 0) const
```

Comment:
```suggestion
    void SpansLocalSpaceSBM(
        std::vector<double>& rSpans,
        IndexType DirectionIndex = 0) const
```
Please follow the [style guide](https://kratosmultiphysics.github.io/Kratos/pages/Kratos/For_Users/Conventions/Style_Guide.html).